### ISSUE TITLE
Enhance github theme for accessibility (minimum contrast)

### ISSUE THREAD

**Issue by oddhack** [rouge-ruby/rouge#1730]
**Is your enhancement request related to a problem? Please describe.**

The github theme has a couple of color / background choices that raise accessibility issues in tools like [ANDI](https://www.ssa.gov/accessibility/andi/help/howtouse.html) - specifically, comment and constant colors in C language highlighting (and probably many other languages) fail the 4.5:1 contrast ratio requirement.

**Describe the solution you'd like**
Update github theme CSS packaged with rouge to satisfy ANDI requirements. This means shifting these two colors a bit to increase contrast relative to the grayish background of the theme, following ANDI recommendations.

**Additional context**
Before capturing screenshots etc., I'd like to ask if this is a change that the project is willing to accept? If so I'll provide more details, and propose a PR if the themeing system is straightforward to modify. If not I'll deal with this by a CSS override in our project. Accessibility issues seem of general use. But I don't understand the full context of where the github theme originated or how important compatibility between the origin, rouge, and other source highlighters is to you.

**Comment by oddhack** [comment]
I'd like this to remain open. We have worked around it by CSS overrides but that's not a great long-term solution (and was very tedious to figure out what the many overrides *were*), and doesn't benefit other users.

**Comment by Bukama** [comment]
It's the same for the rogue default theme used by website (e.g. the JUnit User Guide).

**Comment by jneen** [comment]
You can actually create your own theme without CSS overrides, fwiw.

**Comment by jneen** [comment]
...or use one of a large number of pygments stylesheets https://stylishthemes.github.io/Syntax-Themes/pygments/
